pkg/util/github/github.go

```diff
@@ -193,18 +193,18 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 		tarFile := assetTmpFile[:len(assetTmpFile)-3]
 		err = archive.Ungzip(assetTmpFile, tarFile)
 		if err != nil {
-			return errors.Wrapf(err, "Unable to ungzip %s", assetTmpFile)
+			return errors.Wrapf(err, "Cannot run ungzip %s", assetTmpFile)
 		}
 		err = archive.Untar(tarFile, tmpDir)
 		if err != nil {
-			return errors.Wrapf(err, "Unable to untar %s", tarFile)
+			return errors.Wrapf(err, "Cannot run untar %s", tarFile)
```

Comment:
I am not "running" 'tar' the binary. I "untar". I am running the inverse tart algorithm.